src/branch-manager-service/task-service/presubmit-pr.ts

```diff
@@ -13,22 +20,41 @@ import {setStatusOnGithub} from '../shared/github';
 export const handlePresubmitPrTask = functions
   .pubsub
   .topic('presubmit-pr-task')
-  .onPublish(async message => {
-    const {org, repo, pr, branch, sha} = message.json;
+  .onPublish(async (message: functions.pubsub.Message) => {
+    const {org, repo, pr, branches, sha} = message.json as PresubmitPrTaskData;
     const url = `https://presubmit-service-dot-branch-manager.appspot.com/check_pr`;
-    const config = {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json',
-      },
-      body: JSON.stringify({org, repo, pr, branch, sha})
+    const createConfig = (branch: string) => {
```

Comment:
We should somehow rename this function to imply that this will be the request to the presubmit check endpoint.

Reply:
I went with `config` since `config` is the name of the parameter in `fetch()`.

Jeremy is going to ask that I move the function outside of the `onPublish` callback function anyway, so renaming will also probably need to happen for that.

Reply:
Yeah something like `requestConfig` would be already good enough 😄